### DISCUSSION TITLE
analysing-5-years-of-amazon-kindle-reading/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-personal#23] [Comments]
# analysing-5-years-of-amazon-kindle-reading/

One of my favourite possessions that I use almost every day is my Kindle Paperwhite 3 from 2016. I purchased the Paperwhite 5 recently (comparison coming soon!), and decided to analyse the hundreds of books read on my previous Kindle before I say goodbye to it!

https://jakelee.co.uk/analysing-5-years-of-amazon-kindle-reading/

**Comment by erens-basement** [comment]
Hey just found this post. Will this work with sideloaded sendtokindle books? I know Amazon strips the metadata away from the books so I was wondering if I could still get any meaningful data out of it.

**Reply by JakeSteam** [reply]
Yep! 99% of my books are side loaded, it isn't a problem since the reading data is still tracked.

**Comment by zackmo** [comment]
Great stuff. I think the side loaded content lives in the <code>Kindle.KindleDocs.DocumentMetadata.csv</code> file while anything you've purchased through the Kindle store lives in the <code>Digital.SeriesContent.Relation.2/BookRelation.csv</code> file?

That's how mine was set up. As far as I can tell, the side loaded content usually has the longer 32 digit IDs while the Kindle purchased content is listed by it's 10 digit ASIN.

**Reply by willdxn** [reply]
I think `BookRelation.csv` is just referring to books that Amazon has designated as part of a *series *as it doesn't contain all of the side-loaded but standalone books that I've purchased
